core/templates/pages/contributor-dashboard-page/services/contribution-and-review-backend-api.service.spec.ts

```diff
@@ -74,7 +74,9 @@ describe('Contribution and review backend API service', () => {
       carbas.fetchSuggestionsAsync(
         'SUBMITTED_QUESTION_SUGGESTIONS',
         AppConstants.OPPORTUNITIES_PAGE_SIZE,
-        0, 'All'
+        0,
+        undefined,
```

Comment:
In general we try not to allow undefined -- either the arg is provided or it is not, and if it is not we explicitly provide null. Could you please update accordingly? 

I would actually suggest making it a mandatory argument if possible and dropping all the "optional"-ity in the backend, i.e. we always provide a sort key, since that will simplify things. (There doesn't really seem to be a use case where a sort key isn't needed, right?)